### ISSUE TITLE
[A11yAudit] Insufficient color contrast for non-text elements - Modals

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/Backlog#1336]
**User Story**
As Spark we want to 

**Notes:**
Paciello Audit on Spark Design System
QLC-4 Modals
Insufficient color contrast for non-text elements

ISSUE DESCRIPTION
The level of contrast between foreground and background colors (or, more generally, "adjacent" colors of elements that must be distinguishable) affects accessibility and usability. More contrast makes it easier to distinguish important content and graphical elements. Without sufficient contrast, users may find it difficult, or be completely unable, to correctly perceive and understand important content (such as individual important parts of a chart or graph, or controls denoted purely through graphical elements). This is particularly important for people with visual impairments that affect visual acuity or color perception.

We encountered a small number of color contrast issues where graphical elements lack sufficient contrast. Typical examples are shown [audit]

RECOMMENDATIONS
Provide enough contrast between the foreground color and background color (or, more generally, "adjacent" colors that need to be distinguishable in order to correctly perceive/understand the graphical elements or their component parts) so that people who are color-blind or with moderately low vision can sufficiently distinguish important content. The contrast ratio should be at least 3:1.

**AC**

**Comment by ssabo1** [comment]
Research to confirm placeholder text color contrst.

**Comment by ssabo1** [comment]
This needs to be broken down further.

**Comment by ssabo1** [comment]
Confirmed: This has been added to Spark Design System Team Backlog.
